doc/code/WatchFaces.md

```diff
@@ -0,0 +1,193 @@
+                                                     motionController);
+}
+```
+
+Add to ~line 40 an additional case to select the new screen
+
+```cpp
+        case 3:
+          return WatchFaceTerminalScreen();
+          break;
+        case 4:
+          return WatchFaceAccurateWordsScreen();
+          break;
+      }
+      return WatchFaceDigitalScreen();
+```
+
+Add these reference to the head files
+
+./src/displayapp/screens/Clock.h:
+
+```cpp
+        std::unique_ptr<Screen> WatchFacePineTimeStyleScreen();
+        std::unique_ptr<Screen> WatchFaceTerminalScreen();
+        std::unique_ptr<Screen> WatchFaceAccurateWordsScreen();
+      };
+}
+```
+
+Amend the setting file to allow this watch face to be selected, by incrementing  the number of options and adding the name.
+
+./src/displayapp/screens/settings/SettingWatchFace.h
+
+```cpp
+      private:
+        static constexpr std::array<const char*, 5> options = {"Digital face", "Analog face", "PineTimeStyle", "Terminal", "Words"};
+        Controllers::Settings& settingsController;
+```
+
+./src/displayapp/screens/settings/SettingWatchFace.cpp
+
+```cpp
+constexpr std::array<const char*, 5> SettingWatchFace::options;
+
+```
+
+
+Add a referenace to WatchFaceAccurateWords.cpp 
+
+./src/CMakeLists.txt:
+
+```cpp
+        ## Watch faces
+        displayapp/icons/bg_clock.c
+        displayapp/screens/WatchFaceAnalog.cpp
+        displayapp/screens/WatchFaceDigital.cpp
+        displayapp/screens/WatchFaceTerminal.cpp
+        displayapp/screens/WatchFacePineTimeStyle.cpp
+        displayapp/screens/WatchFaceAccurateWords.cpp
+```
+
+Then finally amened WatchFaceAccurateWords.cpp and .h with the new names rather than the old WatchFaceDigital
+
+./src/displayapp/screens/WatchFaceAccurateWords.cpp:
+
+```cpp
+#include "displayapp/screens/WatchFaceAccurateWords.h
+...
+WatchFaceAccurateWords::WatchFaceAccurateWords(DisplayApp* app,
+                                   Controllers::DateTime& dateTimeController,
+...
+WatchFaceAccurateWords::~WatchFaceAccurateWords() {
+  lv_task_del(taskRefresh);
+...
+void WatchFaceAccurateWords::Refresh() {
+  statusIcons.Update();
+```
+
+./src/displayapp/screens/WatchFaceAccurateWords.cpp:
+
+```cpp
+...
+  namespace Applications {
+    namespace Screens {
+
+      class WatchFaceAccurateWords : public Screen {
+      public:
+        WatchFaceAccurateWords(DisplayApp* app,
+                         Controllers::DateTime& dateTimeController,
+                         Controllers::Battery& batteryController,
+                         Controllers::Ble& bleController,
+                         Controllers::NotificationManager& notificatioManager,
+                         Controllers::Settings& settingsController,
+                         Controllers::HeartRateController& heartRateController,
+                         Controllers::MotionController& motionController);
+        ~WatchFaceAccurateWords() override;
+...
+```
+
+
+## Test Copied Watch Face
```

Comment:
I don't think that the watchface documentation is the right place for instructions on how to use InfiniSim. It has its own repository with own documentation. Recursive docs lead to outdated docs.